extractor.js

```diff
@@ -0,0 +1,44 @@
+const { existsSync, mkdirSync, readFileSync, writeFileSync } = require('fs');
```

Comment:
Is there a reason why there is a custom extractor script, instead of using https://github.com/i18next/i18next-scanner?

Reply:
Hi @Markionium 
After doing the extractor for different applications I got the conclusion it is better to define our own extractor using a customised regex pattern. I used i18next-scanner for our first attempt and it worked for that case. However it is a bit limited and would not work for other cases. For every app I have been refactoring I have to customise something because they follow different implementations/structures.

Reply:
It would make sense to create a generic dhis2 extractor and put it on npm as a package, so it can be reused by all the other apps you're putting this in. I'm guessing this is already part of your long term plan?

Reply:
I agree that it would be the ideal, however it is not so linear. It would be possible if all applications were following the same structure and implementation strategy what is not true at the moment. 

In the end, I think it will only be possible when all the frontend developers and applications agreed upon some common code conventions. Then the extractor could be built based on that. 

In the regard the long term plans, it is better to talk with @jason-p-pickering and Samuel.